Cython/Includes/cpython/datetime.pxd

```diff
@@ -1,27 +1,99 @@
 from cpython.object cimport PyObject
+from cpython.version cimport PY_VERSION_HEX
 
 cdef extern from "Python.h":
     ctypedef struct PyTypeObject:
         pass
 
 cdef extern from "datetime.h":
     """
-    #if PY_MAJOR_VERSION < 3 && !defined(PyDateTime_DELTA_GET_DAYS)
-    #define PyDateTime_DELTA_GET_DAYS(o)         (((PyDateTime_Delta*)o)->days)
+    /* Backport for Python < 3.3 */
+    #ifndef PyDateTime_DELTA_GET_DAYS
```

Comment:
I'd rather keep a maximum version in here so that this code doesn't break if anyone ever decides to turn these into inline functions or so. It's easy to control the past – we rarely succeed in controlling the future.

Just use `PY_VERSION_HEX` with the appropriate CPython release.